vimiv/gui/thumbnail.py

```diff
@@ -143,6 +144,75 @@ def n_rows(self) -> int:
         """Return the number of rows."""
         return math.ceil(self.count() / self.n_columns())
 
+    def _first_index(self) -> int:
```

Comment:
Nitpick, but consider renaming to `_first_rendered_index` for clarification. Same for `_last` later.